config/routes.rb

```diff
@@ -26,4 +26,15 @@
   end
 
   resource :contact, only: :create
+
+  namespace :admin do
+    root 'pages#home'
+    resources :advisers, only: [:index, :show]
+    resources :firms, only: [:index, :show] do
+      resources :advisers, only: :index
+    end
+    resources :lookup_advisers, only: :index, path: 'lookup-advisers'
```

Comment:
Once these are namespace we won't need to explicitly set the `path` either. I know you're aware just putting it down for completion sake :smile: